src/retest/main.c

```diff
@@ -7,8 +7,6 @@
 #if __linux__
 /* apparently you need this for Linux, and it breaks macOS */
 #  define _POSIX_C_SOURCE 200809L
-#else
-#  define _POSIX_C_SOURCE 2
```

Comment:
If it interferes with macos, we should probably have an `#elif !defined(__APPLE__)`.

Unless we don't need it for any other arch.